src/main/java/com/grasp/controller/TutorController.java

```diff
@@ -26,4 +28,29 @@ public TutorController(TutorService tutorService) {
     public ResponseEntity<List<User>> getAllTutors() {
         return new ResponseEntity<>(tutorService.getAllTutors(), HttpStatus.OK);
     }
+
+    @RequestMapping(value = "/byCourse/{courseCode}", method = RequestMethod.GET)
+    public ResponseEntity<List<User>> getTutorsByCourseCode(@PathVariable("courseCode") String courseCode) {
```

Comment:
Shouldn't return a bare list. It's a security risk as it's valid js code. Instead wrap it in a separate DTO.